tests/components/yale/test_config_flow.py

```diff
@@ -0,0 +1,207 @@
+"""Test the yale config flow."""
```

Comment:
I'm missing a test where we try to add the same account twice

Reply:
- [x] yale 06195fcb7077c97a0ac12fa864280be58fd5ba7a
- [x] august does not apply